src/fc-menuitem.ts

```
@@ -159,13 +162,13 @@ export class FcMenuItemElement extends ThemableMixin(LitElement) {
 		});
 		this.addEventListener('click', (event) => {
-		let myEvent = new CustomEvent('menuitem-clicked-event', { 
-			detail: { message: 'Menu item clicked.' },
-			bubbles: true, 
+			let myEvent = new CustomEvent('menuitem-clicked-event', {
+				detail: { message: 'Menu item clicked.' },
+				bubbles: true,
 			composed: true });
-		this.dispatchEvent(myEvent);
+			this.dispatchEvent(myEvent);
 		eval( this.onMenuItemClicked );
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)